docs/core/porting/libraries.md

```diff
@@ -193,6 +193,7 @@ Note that all of the .NET Framework and .NET Standard targets have names recogni
 .NET Framework 4.6   --> NET46
 .NET Framework 4.6.1 --> NET461
 .NET Framework 4.6.2 --> NET462
+.NET Framework 4.7   --> NET47
```

Comment:
is this the same info that is being shown on #2606? If so, I'd probably move the content to an include file so once it changes, it changes across the board for all topics showing the same info. This is what I did for the ,net standard table.